systemverilog-plugin/UhdmAst.cc

```diff
@@ -2437,6 +2437,13 @@ void UhdmAst::process_operation()
     case vpiAssignmentPatternOp:
         process_assignment_pattern_op();
         break;
+    case vpiWildEqOp:
+    case vpiWildNeqOp: {
+        const uhdm_handle *const handle = (const uhdm_handle *)obj_h;
+        const UHDM::BaseClass *const object = (const UHDM::BaseClass *)handle->object;
```

Comment:
Same as https://github.com/chipsalliance/yosys-f4pga-plugins/pull/307#discussion_r857478705  - let's pass `object` here.

Reply:
Fixed here, let's merge this first and then rebase #307